include/sys/zio_compress.h

```diff
@@ -51,15 +51,75 @@ enum zio_compress {
 	ZIO_COMPRESS_GZIP_9,
 	ZIO_COMPRESS_ZLE,
 	ZIO_COMPRESS_LZ4,
+	ZIO_COMPRESS_ZSTD,
```

Comment:
once again, move zstd to the end

Reply:
nope. ZIO_COMPRESS_FUNCTIONS is the terminator for the enum array. eveything after the last entry will be ignored. so after LZ4 is correct